style.css

```diff
@@ -1,10 +1,12 @@
 
 * {
   box-sizing: border-box;
+  overflow-x: hidden;
```

Comment:
- [ ] This recent change that you introduced after the first review is causing visual bugs! 😱 Please remove this `overflow-x` property from the global selector, which is causing the `overflow-y` to be present on most of your elements all over your page. 🤖  For more information about using the `overflow` property, you could refer to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x#values).

![imageedit_6_9564632768](https://user-images.githubusercontent.com/77069011/201484205-25d0871f-b440-49f6-9c61-ce70c3bec16c.png)

Please, consider fixing this! 🤓 🛠️

Reply:
Fixed.